srtcore/stats.h

```diff
@@ -56,7 +56,7 @@ class BytesPackets
 
     BytesPackets(uint64_t bytes, size_t n = 1)
         : m_bytes(bytes)
-        , m_packets(n)
+        , m_packets((uint32_t) n)
```

Comment:
I think a better idea would be to use `uint32_t` type for the constructor argument, but dunno how it will influence the application place.

Reply:
Ok, I made a short research of the call and it looks like almost everywhere the argument is explicitly cast to `size_t`, and where it isn't, the source type isn't `size_t` either (and it's usually a 32-bit type, `int32_t` or `int`). I think this can be safely changed to `uint32_t` and possibly explicitly cast in some places.